libraries/AP_MSP/msp_version.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#if  HAL_MSP_ENABLED
+
+// use betaflight 4.2.0
```

Comment:
Why?

Reply:
right now version is hardcoded to guarantee DJI compatibility